parfive/results.py

```diff
@@ -6,6 +7,9 @@
 
 __all__ = ['Results']
 
+class Error(namedtuple("error", ("filepath_partial", "url", "exception"))):
+    def __repr__(self):
+        return f"{self.url},\n{self.exception}"
```

Comment:
If filename is a string you could add it? Also probably want to do this for str as well.